console/src/components/Services/VoyagerView/VoyagerView.tsx

```diff
@@ -42,12 +42,35 @@ class VoyagerView extends Component<Props, State> {
       body: JSON.stringify({ query }),
     });
 
+  loadWorker = async () => {
+    const url = Endpoints.voyagerWorker;
+    const response = await fetch(url);
```

Comment:
Shouldn't we handle the case when `jsdelivr` is down or we can't fetch the resource?

Reply:
I'm not sure, of how we can handle it. Should we have alternate addresses for fetching the worker?